website/content/api-docs/system/storage/raftautopilot.mdx

```diff
@@ -0,0 +1,164 @@
+---
+layout: api
+page_title: /sys/storage/raft/autopilot - HTTP API
+sidebar_title: <code>/sys/storage/raft/autopilot</code>
+description: |-
+
+  The `/sys/storage/raft/autopilot` endpoints are used to manage raft clusters using autopilot with Vault's Raft storage backend.
+
+---
+
+## Get Cluster State
+
+This endpoint is used to retrieve the raft cluster state.
```

Comment:
Maybe? Or we could repeat it here.

```suggestion
This endpoint is used to retrieve the raft cluster state. See the [docs page](/docs/commands/operator/raft#autopilot-state) for a description of the output.
```

Reply:
I'm confused.  Both pages (this one and the one you link) contain appropriate example output.  What do you feel is missing?

Reply:
We show the output but don't describe it.

Reply:
Ah makes sense.